solidity-flycheck.el

```diff
@@ -212,7 +212,7 @@ no .soliumrc.json is found, `project-roots' is used."
 (defun solidity-flycheck--solc-remappings ()
   (let* ((allow-paths (solidity-flycheck--solc-allow-paths)))
     (->> allow-paths
-         (remove-if-not #'file-exists-p)
+         (cl-remove-if-not #'file-exists-p)
```

Comment:
For me `C-h s remove-if-not` shows

> remove-if-not is an alias for ‘cl-remove-if-not’ in ‘cl.el’.

If you're getting issues, going to approve and merge it anyway.